qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -622,6 +622,6 @@
             gagent_status_cmd = "pgrep qemu-ga"
             gagent_restart_cmd = "pgrep qemu-ga |xargs kill -s 9 &&  ${gagent_start_cmd}"
             Windows:
-                gagent_start_cmd = "cd C:\Program Files\Qemu-ga & start /b qemu-ga.exe -m isa-serial -p COM2 -l c:\qga.log -v"
+                gagent_start_cmd = "cd C:\Program Files\Qemu-ga & start /b qemu-ga.exe -m isa-serial -p COM2 -l c:\qga.log -v > null"
```

Comment:
@vivianQizhu Hi Qianqian, after confirming, normally, 'nul' is the correct spelling, but Windows also accepts 'null' for compatibility reasons. This is a legacy issue, they can both be used to redirect the output of a command to an empty device

Reply:
@6-dehan Well noted, thanks for clarification. Then let's keep it `nul`.

Reply:
okay, got it.

Reply:
Could you explain the reason for redirecting the output to nul ?  IMO, it is not the root cause of this issue, and IMO, the output (session log) is helpful for debugging in case there are any problem.

Reply:
those are useless that printout to the terminal, and cause inconvenience for our automation. plus,  the output should be in our new debug tool log. so we can just redirect them to NUL.

Reply:
What do you mean `debug tool log`, it is a session command and i think it is in the session log and helpful for debug.

And, for the inconvenience,  IMO, as long as automation can get the prompt pattern is enough, it can be resolved with safe mode with a new line,  Is there any other inconvenience i missed ? Thanks!

Reply:
Ah, ignore me,  the command already save log to qga.log , i did not notice it, thanks!